pymysql/tests/test_basic.py

```diff
@@ -91,11 +91,24 @@ def test_integer(self):
         finally:
             c.execute("drop table test_dict")
 
-    def test_blob(self):
+    def test_binary(self):
         """test binary data"""
         data = bytes(bytearray(range(256)) * 4)
         conn = self.connections[0]
         self.safe_create_table(
+            conn, "test_binary", "create table test_binary (b binary)")
+
+        with conn.cursor() as c:
+            c.execute("insert into test_binary (b) values (%s)", (data,))
```

Comment:
Travis test with Python 2.7 fails by "Data truncated for column 'b' at row 1" warning.